app/server/api/credit_transfer_api.py

```diff
@@ -312,7 +312,7 @@ def post(self, credit_transfer_id):
                         transfer_use=transfer_use,
                         transfer_mode=TransferModeEnum.WEB,
                         uuid=uuid,
-                        automatically_resolve_complete=auto_resolve,
+                        automatically_resolve_complete=False,
```

Comment:
If we don't use `automatically_resolve_complete` anymore, should we consider deprecating it?

Reply:
I think out of scope, but we should probably do it in the future IMO

Reply:
ok